tests/tests-filters.php

```diff
@@ -259,7 +259,7 @@ public function test_comments_clauses() {
 	public function test_edd_update_payment_status() {
 		global $wp_filter;
 		$this->assertArrayHasKey( 'edd_complete_purchase', $wp_filter['edd_update_payment_status'][100] );
-		$this->assertArrayHasKey( 'edd_record_status_change', $wp_filter['edd_update_payment_status'][100] );
+		$this->assertArrayHasKey( 'edd_record_order_status_change', $wp_filter['edd_update_payment_status'][100] );
```

Comment:
Nice job finding this. :raised_hands:

Reply:
Oh one minor thing: that second argument needs to be updated too:

```suggestion
		$this->assertArrayHasKey( 'edd_record_order_status_change', $wp_filter['edd_transition_order_status'][100] );
```